### PR TITLE
Registration

### PR DESCRIPTION
**Pull request by benmyerson**
https://github.com/kevinjamesus86/pdxpong-client/issues/5 and then some...

Nav header shows default user icon until user logs in with a provider.
Clicking the username in the nav header takes you to an account section with subpages for account (password/account linking/deleting/etc) and profile (images/nickname/player related stuff) and a link for logging out.

### CODE REVIEW

File: app/src/components/account-account/account-account.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+import angular from 'angular';
+import uiRouter from 'angular-ui-router';
+import templateUrl from './account-account.html';
+
+class AccountAccount {
+    constructor($state, Auth, currentAuth) {
+        this.Auth = Auth;
+        this.$state = $state;
+        console.log(currentAuth);
```
Comment (benmyerson) [inline app/src/components/account-account/account-account.js]: this breaks without a rules change to eslint.js

Comment (kevinjamesus86) [inline app/src/components/account-account/account-account.js]: feel free to turn off the no-console rule. Get's a little tedious adding and removing `$log`

**Comment by benmyerson** [comment]
Goes with https://github.com/benmyerson/pdxpong-api/pull/5

**Comment by kevinjamesus86** [comment]
Dude, I love this. Nice work.

**Comment by benmyerson** [comment]
Not quite ready for primetime, as it turns out.

**Pull request merged**